.gitignore

```diff
@@ -2,6 +2,7 @@
 coverage
 node_modules
 .temp
+.idea
```

Comment:
Hey @clovisdasilvaneto! This should be listed on your `.gitignore_global`.

Reply:
Oh, ok. I'll correct this ;)